### PR TITLE
[ADD] disable_invalid_filters

### PR DESCRIPTION
**Pull request by pedrobaeza**
Avoid errors on migrated databases making a non-destructive disabling.

It analyzes all the existing active filters to check if they are still correct. If not, they are disabled for avoiding errors when clicking on them, or worse, if they are default filters when opening the model/action.

To be run at the base end-migration script for having a general scope. Only assured to work on >= v8.

When merged, I will make PRs for OpenUpgrade versions calling this method on end-migration/deferred step.

@Tecnativa

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Can we merge then?

**Comment by pedrobaeza** [comment]
There's a conflict now due to a merge operation.

**Comment by StefanRijnhart** [comment]
Well I started the merge operation because there was a conflict in the first place.

**Comment by StefanRijnhart** [comment]
But I don't know what went wrong. Maybe you can rebase your own commit onto master and discard my edits?

**Comment by pedrobaeza** [comment]
Done!

File: openupgradelib/openupgrade.py [hunk 1]
```diff
@@ -1670,6 +1671,7 @@ def rename_property(cr, model, old_name, new_name):
         "update ir_property set name=%s where fields_id in %s",
         (new_name, field_ids))
 
+<<<<<<< HEAD
```
Comment (StefanRijnhart) [inline openupgradelib/openupgrade.py]: Conflict markers were merged.

**Comment by pedrobaeza** [comment]
Resubmitted.

**Comment by StefanRijnhart** [comment]
Success!

**Pull request merged**